packages/safe-tools-client/app/templates/schedule.hbs

```diff
@@ -54,7 +54,11 @@
   <section class='safe-tools__dashboard-schedule-content'>
     <ScheduleCollapsePanel @open={{not this.wallet.isConnected}} />
     <div class='temporary-form-container'>
```

Comment:
Probably a good time to get rid of the "temporary form container" now that we have all 3 panels (form and both lists)

Reply:
Updated